src/mixin/java/com/gtnewhorizons/angelica/mixins/late/client/skidzoom/InventoryPlayerMixin.java

```diff
@@ -0,0 +1,18 @@
+package com.gtnewhorizons.angelica.mixins.late.client.skidzoom;
+
+import net.loudcats.wyatt.skidzoom.SkidZoom;
+import net.minecraft.entity.player.InventoryPlayer;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(InventoryPlayer.class)
+public abstract class InventoryPlayerMixin {
+    @Inject(method = "changeCurrentItem", at = @At("HEAD"), cancellable = true)
```

Comment:
I would prefer that you wrap the `int i = Mouse.getEventDWheel();` line in the `Minecraft.runTick()` method and make it return 0 when it's zooming